Mathlib/RingTheory/Polynomial/DegreeLT.lean

```diff
@@ -0,0 +1,191 @@
+/-
+Copyright (c) 2025 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen, Kenny Lau
+-/
+
+import Mathlib.Algebra.Polynomial.Div
+import Mathlib.Algebra.Polynomial.Taylor
+import Mathlib.LinearAlgebra.Determinant
+import Mathlib.LinearAlgebra.Matrix.Block
+import Mathlib.RingTheory.Polynomial.Basic
+
+/-!
+# Polynomials with degree less than `n`
+
+This file contains the properties of the submodule of polynomials of degree less than `n` in a
+(semi)ring `R`, denoted `R[X]_n`.
```

Comment:
This docstring is incorrect, in the sense that plenty of properties of `degreeLT` are already proved in earlier files. Can you come up with a criterion for what results should be put in this file as opposed to an earlier file?

Reply:
That sounds reasonable, yeah